src/EditorFeatures/Test/CommentSelection/CommentUncommentSelectionCommandHandlerTests.cs

```diff
@@ -562,7 +562,7 @@ private static void CommentSelection(ITextView textView, IEnumerable<TextChange>
             commandHandler.CollectEdits(service,
                 textView.Selection.GetSnapshotSpansOnBuffer(textView.TextBuffer), textChanges, trackingSpans, operation);
 
-            AssertEx.SetEqual(expectedChanges, textChanges);
+            Roslyn.Test.Utilities.AssertEx.SetEqual(expectedChanges, textChanges);
```

Comment:
Fallout from adding IVT to from this project to ServicesTestUtilities2 (for the mock navigation stuff).